src/wrappers/SD_TC.h

```diff
@@ -22,25 +22,28 @@ class SD_TC {
   static void deleteInstance();
 
   // instance methods
-  void appendData(const char* header, const char* line);
+  void writeToDataLog(const char* header, const char* line);
   void appendToLog(const char* line);
+  bool countFiles(void (*callWhenFinished)(int));
   bool exists(const char* path);
   bool format();
-  char* getRemoteLogName();
+  uint32_t getRemoteFileSize() {
+    return remoteFileSize;
+  }
+  const char* getRemoteLogName();
   bool listRootToBuffer(void (*callWhenFull)(const char*, bool));
-  bool countFiles(void (*callWhenFinished)(int));
   bool mkdir(const char* path);
   File open(const char* path, oflag_t oflag = 0x00);
   void printRootDirectory();
   bool remove(const char* path);
-  void setRemoteLogName(const char* newFileName);
+  void setRemoteLogName(const char* newFileName = nullptr);
   void todaysDataFileName(char* path, int size);
   void writeToRemoteLog(const char* line);
 
 #if defined(ARDUINO_CI_COMPILATION_MOCKS)
-  char mostRecentHeader[128] = "";
-  char mostRecentLine[128] = "";
-  char mostRecentRemoteEntry[256] = "";
+  char mostRecentDataLogHeader[128] = "";
+  char mostRecentDataLogLine[128] = "";
+  char mostRecentRemoteLogEntry[256] = "";
   void updateAlertFileSizeForTest() {
     updateRemoteFileSize();
   }
```

Comment:
This method appears to be called only by the commented-out lines in SDTest.cpp. Can it be removed and replaced by updateRemoteFileSize()?

Reply:
Speaking of which, is remoteFileSize used anywhere other than tests?